contrib/macdeploy/fancy.plist

```diff
@@ -10,7 +10,7 @@
 		<integer>620</integer>
 	</array>
 	<key>background_picture</key>
-	<string>background.png</string>
+	<string>background.tiff</string>
```

Comment:
tiff is used to include a retina and non-retina background (according to https://developer.apple.com/library/mac/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Optimizing/Optimizing.html)